src/common/config.cc

```diff
@@ -1024,6 +1024,16 @@ void md_config_t::get_config_bl(
   }
 }
 
+std::optional<std::string> md_config_t::get_val_default(std::string_view key)
+{
+  std::string val;
+  const Option *opt = find_option(key);
+  if (opt && (conf_stringify(_get_val_default(*opt), &val) == 0)) {
+    return std::make_optional(std::move(val));
```

Comment:
No problem with this version, but note that
'return val;'
produces the same code.

(if you'd like to verify: https://godbolt.org/z/j4P5xG4fv )

Reply:
Apart from that - LGTM

Reply:
Yes, that's correct (nice tool!). I would prefer retaining the current version for readability.